playbooks/prerequisite.yaml

```diff
@@ -12,3 +12,9 @@
   roles:
   - rhsm-repos
   - prerequisites
+
+- hosts: master
+  gather_facts: no
```

Comment:
One thing to keep in mind here is that the package module makes use of facts, so If for some reason it ends up being possible to call just this play (through tags), then the package task will fail for undefined variables.